src/views/Proposal.vue

```diff
@@ -68,8 +68,21 @@
           <div class="mb-1">
             <b>Token</b>
             <span class="float-right text-white">
-              <Token :space="space.key" class="mr-1" />
-              {{ space.symbol }}
+              <span
+                v-for="(symbol, symbolIndex) of symbols"
+                :key="symbol"
+                class="token-title"
+              >
+                <Token
+                  :space="space.key"
+                  :symbol="symbol"
+                  :show-symbol="true"
+                  class="mx-1"
```

Comment:
This content is aligned in right, there shouldn't be a space in the right side, will be better to remove that class anyway because there is already a space in the " + "
```suggestion
```

Reply:
Applied suggested change